src/Sulu/Bundle/AdminBundle/Resources/js/components/ResourceLocator/README.md

```diff
@@ -0,0 +1,28 @@
+The ResourceLocator component can be used to get a URL from user input in two modes.
+
+* `full`: User can change everything after the initial slash.
+* `leaf`: User just can change the part after the last slash.
```

Comment:
I wouldn't use a bullet list here... It might not be completely clear which rendered example below belongs to what. I would write it somehow like this instead:

```
In the `full` mode the user can change the entire URL except for the leading slash:

// First example

In the `leaf` mode the user is only capable of editing the part after the last slash:

// second example
```